kernel/os/src/arch/cortex_m33/os_fault.c

```diff
@@ -213,6 +215,8 @@ os_default_irq(struct trap_frame *tf)
     uint32_t orig_sp;
 #endif
 
+    memmask_clear_sensitive_area();
```

Comment:
similar to `log_reboot` this should be made conditional.

Reply:
Added a syscfg condition to enable the clearing feature